frontend/src/components/ViewTrips/trip.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+/**
+ * Returns the title of the trip associated with the Trip document data
+ * `tripObj`.
+ *
+ * @param {firebase.firestore.DocumentData} tripObj Object containing the fields
+ *    and values for a Trip document.
+ * @return Title of the trip (if it exists).
+ */
+function createTitleElement(tripObj) {
+  try {
+    if('name' in tripObj) {
+      return tripObj.name;
+    }
+    throw new Error(`Property 'name' is not defined in 'tripObj.'`);
```

Comment:
I assume you handle the error here manually because if 'name' is not in tripObj it will still let you get tripObj.name (but it will be null or something)?

Reply:
Yeah it returns undefined which I don't want so that is why I threw that error manually to be caught by the following catch.